src/test/java/org/heigit/ohsome/ohsomeapi/controller/GetControllerTest.java

```diff
@@ -681,6 +681,30 @@ public void getUsersCountDensityGroupByBoundaryTest() {
         expectedValue * deltaPercentage);
   }
 
+  @Test
+  public void getUsersCountDensityByContributionTypeTest() {
+    double expectedValue = 0.54;
+    TestRestTemplate restTemplate = new TestRestTemplate();
+    ResponseEntity<JsonNode> response = restTemplate.getForEntity(server + port
+        + "/users/count/density?bboxes=8.67,49.39,8.71,49.43&"
+        + "filter=type:way and natural=*&contributionType=creation&"
+        + "time=2014-01-01/2017-01-01", JsonNode.class);
+    double result = response.getBody().get("result").get(0).get("value").asDouble();
+    assertEquals(expectedValue, result, deltaPercentage);
+  }
+
+  @Test
+  public void getUsersCountGroupByKeyByContributionTypeTest() {
```

Comment:
to make the naming slightly more similar to the following "contributions tests":

```suggestion
  public void getUsersCountGroupByKeyFilteredByContributionTypeTest() {
```